specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2018-08-01/ResourceHealth.json

```diff
@@ -1095,6 +1128,156 @@
+          "description": "The impacted region slug id."
+        },
+        "name": {
+          "type": "string",
+          "description": "The impacted region name."
+        }
+      }
+    },
+    "emergingIssueImpact": {
+      "type": "object",
+      "description": "Object of the emerging issue impact.",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The impacted service slug id."
+        },
+        "name": {
+          "type": "string",
+          "description": "The impacted service name."
+        },
+        "regions": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/impactedRegion"
+          },
+          "description": "The list of impacted regions for corresponding emerging issues."
+        }
+      }
+    },
+    "statusActiveEvent": {
+      "description": "Active event type of emerging issue.",
+      "properties": {
+        "title": {
+          "type": "string",
+          "description": "The active event title."
+        },
+        "description": {
+          "type": "string",
+          "description": "The details of active event."
+        },
+        "trackingId": {
+          "type": "string",
+          "description": "The tracking id of this active event."
+        },
+        "startTime": {
+          "type": "string",
+          "description": "The impact start time on this active event.",
+          "format": "date-time"
+        },
+        "cloud": {
+          "type": "string",
+          "description": "The cloud type of this active event."
+        },
+        "severity": {
+          "type": "string",
+          "description": "The severity level of this active event."
+        },
+        "stage": {
+          "type": "string",
+          "description": "The stage of this active event, eg. Active."
+        },
+        "published": {
+          "type": "boolean",
+          "description": "The boolean value of this active event if published or not."
+        },
+        "lastModifiedTime": {
+          "type": "string",
+          "description": "The last time modified on this banner.",
+          "format": "date-time"
+        },
+        "impact": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/emergingIssueImpact"
+          },
+          "description": "The list of emerging issues impacts."
+        }
+      }
+    },
+    "emergingIssuesGetResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/emergingIssue"
+          },
+          "description": "The list of emerging issues."
+        }
+      },
+      "required": [
+        "value"
+      ],
+      "description": "The Get EmergingIssues operation response."
+    },
+    "emergingIssue": {
+      "type": "object",
+      "description": "On-going emerging issue from azure status.",
+      "properties": {
+        "id": {
```

Comment:
Even proxy resources like this should typically have a resource "id", "name", and "type" in their payload.  Please also change "id" to "correlationId" or similar to not be confusing

Reply:
Make sense. I'll remove this id then. actually it is not necessary to expose this id to user.